crawl-ref/source/dat/des/serial/subway.des

```diff
@@ -0,0 +1,60 @@
+# A small vault allowing fast travel around a level. Unlike other transporter
+# vaults, you don't see where you end up, but there's no big guaranteed hazards
+# at the other end either. So you'll probably be fine!
+# Each "subway station" is labelled by the material that appears twice, in
+# opposite corners. Each "exit" is labelled by the material that appears in the
+# corner next to that transporter. For example, if you exit through the
+# transporter next to the crystal, you come out at the "crystal" station; that
+# is, the one with two crystal corners. It's easy. You'll figure it out.
+
+NAME:   serial_nicolae_subway
+DEPTH:  D:8-, Depths, Elf, Shoals, Snake, Zot
+TAGS:   allow_dup luniq_serial
```

Comment:
Probably best to address the balance concern with weighting. I agree you'd not want this showing up very often, and I don't think it actually needs `allow_dup`, but it just being a minivault alone means it will be seen quite infrequently in earlier levels. Random stuff in level gen does let you split up monsters very easily, of course stairs/hatches are the main culprit, and while it's true there's an additional delay for stairs, this is not something that reduces the effectiveness of that tactic by some very large factor to the point where we meticulously control stair placement relative to layout and monster generation on every level.

It would be fine to give it half weight, although I don't think that's actually necessary; I would recommend removing `allow_dup` though. Nice to see some use of transporters beyond the usual self-contained challenge vault, and I think the core idea of this vault is fine as not-too-frequent encounter.